.github/workflows/python-tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
   python-tests-build-image:
     runs-on: ubuntu-latest
     container:
-      image: public.ecr.aws/opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2
+      image: public.ecr.aws/opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3
```

Comment:
I never observed that we use docker image for this :)

Reply:
I added, due to last time we broke on the `requests` pkg upgrade: 
* https://github.com/opensearch-project/opensearch-build/pull/3559